src/commands/deploy.ts

```diff
@@ -30,6 +30,10 @@ export default class Deploy extends Command {
     "frontend-refs-path": flags.string({
       default: "./frontend/src/refs.terrain.json",
     }),
+    "arm64": flags.boolean({
+      description: "use rust-optimizer-arm64 for optimization. Not recommended for production, but it will optimize quicker on arm64 hardware during development.",
+      default: false,
+    }),
```

Comment:
Is there maybe a way to throw an error or warn when the user tries to use this build with production deployment combined?